libmscore/property.cpp

```diff
@@ -275,8 +276,8 @@ static constexpr PropertyMetaData propertyList[] = {
       { Pid::FRAME_TYPE,              false, "frameType",             P_TYPE::INT,                 DUMMY_QT_TRANSLATE_NOOP("propertyName", "frame type")       },
       { Pid::FRAME_WIDTH,             false, "frameWidth",            P_TYPE::SPATIUM,             DUMMY_QT_TRANSLATE_NOOP("propertyName", "frame width")      },
       { Pid::FRAME_PADDING,           false, "framePadding",          P_TYPE::SPATIUM,             DUMMY_QT_TRANSLATE_NOOP("propertyName", "frame padding")    },
-      { Pid::FRAME_ROUND,             false, "frameRound",            P_TYPE::INT,                 DUMMY_QT_TRANSLATE_NOOP("propertyName", "frame round")      },
 //200
```

Comment:
I don't know if these numbers are still up to date, nor if they are useful at all, and I don't like killing the blame like that, so could someone tell me if I should revert that change?

Reply:
What is the value of `Pid::FRAME_ROUND`?

Reply:
OK, no, it is 203 before your change 204 after, so it now would beling to before `Pid::FONT_STYLE,`

Reply:
Ok, should I update it, or remove it?

Reply:
Please update

Reply:
done